client/actions/planning/notifications.ts

```diff
@@ -301,13 +302,20 @@ const udpateAssignment = (planningId) => (
         }
 
         const planningItem = selectors.planning.storedPlannings(getState())[planningId];
+        const promiseResponses = [];
 
         get(planningItem, 'coverages', []).forEach((cov) => {
             if (get(cov, 'assigned_to.assignment_id')) {
-                dispatch(assignments.api.fetchAssignmentById(cov.assigned_to.assignment_id, true));
-                dispatch(assignments.api.fetchAssignmentHistory({_id: cov.assigned_to.assignment_id}));
+                promiseResponses.push(
+                    dispatch(assignments.api.fetchAssignmentById(cov.assigned_to.assignment_id, true))
+                );
+                promiseResponses.push(
+                    dispatch(assignments.api.fetchAssignmentHistory({_id: cov.assigned_to.assignment_id}))
+                );
             }
         });
+
+        return Promise.all(promiseResponses);
```

Comment:
`promises` would be a better name. Promise "response" I would understand as a resolved value.